mishmash.js

```diff
@@ -33,16 +45,17 @@ export const createMishmash = () => {
     $('.content--border').addClass('mishmash')
 
     $('.ingredients.mishmash').click(event => {
+
         if ($(event.target).hasClass('selected')) {
             $(event.target).removeClass('selected')
 
-            const nameIngredients = $(event.target).text()
-            
-           selectedIngredients = selectedIngredients.filter(name => name !== nameIngredients)
-           
+            const ingredientUUID = $(event.target).children().attr('data-id')
```

Comment:
again in this function you are looking for same element 3 times.
`$(event.target)` will look for it in DOM